crates/evm/evm/src/inspectors/mod.rs

```diff
@@ -7,6 +7,9 @@ pub use foundry_evm_traces::{StackSnapshotType, TracingInspector, TracingInspect
 
 pub use revm_inspectors::access_list::AccessListInspector;
 
+mod custom_printer;
+pub use custom_printer::CustomPrintTracer;
```

Comment:
Note: this was re-introduced in https://github.com/foundry-rs/foundry/pull/7716, implemented now inside of Foundry, to be followed up in https://github.com/foundry-rs/foundry/issues/10329